nft-market/components/Mint/Mint.tsx

```diff
@@ -0,0 +1,67 @@
+import { mintAssetToNft } from "@/utils/mint";
```

Comment:
folders should start with lowercase letter and components (so .tsx files) should start with capital

Reply:
Updated the names of folders and components